wfe/web-front-end.go

```diff
@@ -778,20 +778,18 @@ func (wfe *WebFrontEndImpl) NewCertificate(response http.ResponseWriter, request
 		wfe.sendError(response, respMsg, err, respCode)
 		return
 	}
-	logEvent.Requester = reg.ID
-	logEvent.Contacts = reg.Contact
 	// Any version of the agreement is acceptable here. Version match is enforced in
 	// wfe.Registration when agreeing the first time. Agreement updates happen
 	// by mailing subscribers and don't require a registration update.
 	if reg.Agreement == "" {
-		logEvent.Error = "Must agree to subscriber agreement before any further actions"
-		wfe.sendError(response, logEvent.Error, nil, http.StatusForbidden)
+		logEvent.AddError("Must agree to subscriber agreement before any further actions")
+		wfe.sendError(response, "Must agree to subscriber agreement before any further actions", nil, http.StatusForbidden)
```

Comment:
Would it make sense to just pass the `requestEvent` to `sendError()` and avoid all this repetition?